test/k8sT/Chaos.go

```diff
@@ -77,15 +77,26 @@ var _ = Describe("K8sChaosTest", func() {
 
 		for _, pod := range pods {
 			for _, ip := range dsPods {
+				By("\tPinging test-ds service pod with IP %s from client pod %s", ip, pod)
 				res := kubectl.ExecPodCmd(
 					helpers.DefaultNamespace, pod, helpers.Ping(ip))
 				log.Debugf("Pod %s ping %v", pod, ip)
 				ExpectWithOffset(1, res).To(helpers.CMDSuccess(),
 					"Cannot ping from %q to %q", pod, ip)
 
-				err = kubectl.WaitForKubeDNSEntry(testDSService)
+				By("\tWaiting for kube-dns entry for service testds-service")
+				err = kubectl.WaitForKubeDNSEntry(testDSService, helpers.DefaultNamespace)
```

Comment:
Can this be out of the loop? We tested for 6 different pods, and per 2. That can be out of the loop no?

(Worried about the time that this takes)

Reply:
Good idea.

Reply:
Will be fixed in next upload.